es.hjson

```diff
@@ -674,23 +674,23 @@
   "misc.huh.main": "<b>eh</b>"
 
   "misc.judge.main.truth": "%s<b> está diciendo la verdad</b>"
-  "misc.judge.main.lying": "%s<b> está mintiendo!</b>"
+  "misc.judge.main.lying": "¡%s<b> está mintiendo!</b>"
   "misc.judge.CalsiBot": "<i>Estoy más allá del bien y del mal</i>"
 
   "misc.coin.main.heads": "%s lanzó una moneda!\n\n<b>¡Son cabezas!</b>"
   "misc.coin.main.tails": "%s lanzó una moneda!\n\n<b>¡Son colas!</b>"
 
-  "misc.covfefe.tooLong": "<b>❌ Demasiado largo. ☕😐</b>"
-  "misc.covfefe.replyHint": "<b>¡Responda al mensaje que desea cofectar!</b>"
+  "misc.covfefe.tooLong": "<b>❌ Demasiado tiempo. ☕😐</b>"
+  "misc.covfefe.replyHint": "<b>¡Responda al mensaje que deseas covfefe!</b>"
 
-  "misc.spurdo.tooLong": "<b>❌ too lonk :dd 🥺</b>"
-  "misc.spurdo.replyHint": "<b>responda a de meszage que desea spurdo-fy :DD</b>"
+  "misc.spurdo.tooLong": "<b>❌ Demasiado tiempo :dd 🥺</b>"
+  "misc.spurdo.replyHint": "<b>¡Responda al mensaje que desea provocar o estimular! :DD</b>"

```

Comment:
Both of these must remain in lowercase.

And we should add back "spurdo-fy", provocar o estimular is too generic:
https://knowyourmeme.com/memes/spurdo-sparde